src/redux/books/api/deleteBook.js

```diff
@@ -0,0 +1,22 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import url from '../../../http-common';
+import { removeBook } from '../books';
+
+const DELETE_BOOK = 'books/DELETE_BOOK';
+
+const deleteBook = createAsyncThunk(DELETE_BOOK, async (id, thunkAPI) => {
+  const response = await fetch(`${url}/${id}`, {
```

Comment:
- [ ] Nice work so far 👍🏿 on your project. I can see you are using the `fetch`  method for your API calls. But as per project requirement you are to use Axios for the API call instead of  fetch method